interface/frontend/src/components/open-image/ImageSeries.vue

```diff
@@ -8,21 +8,47 @@
             Open imagery
           </div>
           <div class="card-block">
-            <template v-if="availableSeries.length">
+            <template v-if="selected">
+              <table class="table table-bordered table-condensed">
+                <thead>
+                <tr>
+                  <th>key</th>
+                  <th>value</th>
+                </tr>
+                </thead>
+                <tbody>
+                <tr v-for="(item, key, index) in selected">
+                  <td>{{ key }}</td>
+                  <td><small>{{ item }}</small></td>
+                </tr>
+                </tbody>
+              </table>
               <ul>
-                <li v-for="series in availableSeries" :key="series.series_instance_uid">
-                  <a href="#" @click="selectSeries(series)">{{ series.series_instance_uid }}</a>
-                  <span v-if="series == selected">&larr;</span>
+                <li v-for="el in cases" :key="el.created">
+                  <a href="#" @click="selectCase(el)">{{ el.created }}</a>
+                  <span v-if="el == selectedCase">&larr;</span>
                 </li>
               </ul>
+              <button @click="createCase(selected)"
+                      class="btn btn-primary float-left case-btn">
+                Create new case
+              </button>
+              <a href="#/report-and-export" class="btn btn-primary float-left case-btn">Start case</a>
```

Comment:
As far as I understood from [this comment](https://github.com/concept-to-clinic/concept-to-clinic/pull/236#issuecomment-346512765) we should have multiple cases running same time. So there is no need to have **Start case** button. At least until there is no any role for it :)